mibs/webpower-smart2/USHA-MIB

```diff
@@ -0,0 +1,2620 @@
+-- Ingrasys USHA MIB
```

Comment:
Hi @Npeca75 Seems from the file that the original editor of the MIB is Ingrasys. So I would suggest to rename the fil to `mibs/ingrasys/USHA-MIB`

Reply:
> Hi @Npeca75 Seems from the file that the original editor of the MIB is Ingrasys. So I would suggest to rename the fil to `mibs/ingrasys/USHA-MIB`

hi @PipoCanaja 
Done !